src/Renderer/LayeredMaterial.js

```diff
@@ -394,26 +391,41 @@ LayeredMaterial.prototype.isLayerTypeDownscaled = function isLayerTypeDownscaled
 };
 
 LayeredMaterial.prototype.getColorLayerLevelById = function getColorLayerLevelById(colorLayerId) {
-    let index = this.indexOfColorLayer(colorLayerId);
-    // TODO: hiding the "colorLayerId is invalid", it's problem, needs new PR
+    const index = this.indexOfColorLayer(colorLayerId);
     if (index === -1) {
-        index = 0;
+        return -1;
```

Comment:
Then `return EMPTY_TEXTURE_ZOOM`? My point is that we shouldn't return a magic number.